src/main/java/edu/kit/compiler/optimizations/WorklistFiller.java

```diff
@@ -0,0 +1,26 @@
+package edu.kit.compiler.optimizations;
+
+import edu.kit.compiler.io.Worklist;
+import firm.nodes.Node;
+import firm.nodes.NodeVisitor;
+
+/**
+ * Firm node visitor that inserts all visited nodes in the given Worklist.
+ */
+public class WorklistFiller extends NodeVisitor.Default {
```

Comment:
Same as for `NodeCollector`, there are visitors for filling data structures in `optimizations.Util`.

Reply:
Removed